packages/suite/src/views/settings/index.tsx

```diff
@@ -382,12 +394,14 @@ const Settings = () => {
                                         ),
                                     }}
                                 />
-                                {!['checking', 'not-available'].includes(desktopUpdate.state) &&
+                                {(desktopUpdate.state === 'available' ||
+                                    desktopUpdate.state === 'downloading' ||
+                                    desktopUpdate.state === 'ready') &&
```

Comment:
It's definitely matter of taste, feel free to ignore this comment, but do you really find multiple `||` more readable than: 
```suggestion
                                {[ 'available', 'downloading', 'ready'].includes( desktopUpdate.state) &&
```
?

Reply:
ok, it's because of the typescript...